tailcfg/tailcfg.go

```diff
@@ -733,7 +733,38 @@ type NetInfo struct {
 	// This should only be updated rarely, or when there's a
 	// material change, as any change here also gets uploaded to
 	// the control plane.
-	DERPLatency map[string]float64 `json:",omitempty"`
+	// Deprecated; use DERPLatencyV4 and DERPLatencyV6 instead.
+	DERPLatency   map[string]float64 `json:",omitempty"`
+	DERPLatencyV4 map[int]float64    `json:",omitempty"`
+	DERPLatencyV6 map[int]float64    `json:",omitempty"`
+
+	// a UDP STUN round trip completed
+	UDP bool `json:",omitempty"`
+
+	// an IPv6 STUN round trip completed
+	IPv6 bool `json:",omitempty"`
+
+	// an IPv4 STUN round trip completed
+	IPv4 bool `json:",omitempty"`
+
+	// an IPv6 packet was able to be sent
+	IPv6CanSend bool `json:",omitempty"`
+
+	// an IPv4 packet was able to be sent
+	IPv4CanSend bool `json:",omitempty"`
+
+	// an ICMPv4 round trip completed
+	ICMPv4 bool
```

Comment:
Is there no IPv6 version of this?

Reply:
Tailscale doesn't send ICMPv6, and so they don't include it in the report.